www/_posts/2017-03-30-screen-orientation-plugin.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: post
+author:
+    name: Vishal Mishra
+    url: https://twitter.com/tweetsbymishra
+title:  "Screen Orientation Plugin"
+categories: news
+tags: cordova screen orientation
+---
+Originally posted at this [PhoneGap blog](https://blog.phonegap.com/cordova-screen-orientation-plugin-fabeba30a4c4)
+
+## Cordova Screen Orientation Plugin
+
+The new version(2.0.0) of the cordova screen orientation plugin was released recently. The purpose of the plugin is to set/lock the screen orientation in a common way for all the major platforms. This new version conforms with the specifications mentioned in the [W3C Screen Orientation API](https://www.w3.org/TR/screen-orientation/), currently in open draft.
+
+The plugin incorporates these major updates:
+
+1. Locking/unlocking the screen orientation.
+2. Accessing the current screen orientation.
+3. Addition of ‘onchange’ event handler to the screen.orientation object.
+4. A Demo App.
+
+### Locking/unlocking the screen orientation
+
+The screen orientation can be locked to a particular orientation, say, for example, landscape, using :
+
+```js
+screen.orientation.lock(‘landscape’).then(function success() {
```

Comment:
`‘landscape’` does it make sense to replace it to standard quote chars? `'landscape'`